src/sentry/workflow_engine/handlers/condition/slow_condition_query_handlers.py

```diff
@@ -164,13 +230,21 @@ def get_rate_bulk(
         if comparison_interval:
             current_time -= comparison_interval
         start, end = self.get_query_window(end=current_time, duration=duration)
+        conditions = []
+
+        if filters:
+            for filter in filters:
+                snuba_condition = self.convert_filter_to_snuba_condition(filter)
+                if snuba_condition:
+                    conditions.append(snuba_condition)
 
         with self.disable_consistent_snuba_mode(duration):
             result = self.batch_query(
                 group_ids=group_ids,
                 start=start,
                 end=end,
                 environment_id=environment_id,
+                conditions=conditions or None,
```

Comment:
i start using the name `conditions` for the function arguments after the `filters` have been converted to snuba conditions. other naming suggestions would be appreciated

Reply:
name makes sense to me